src/Microsoft.OData.Core/ODataBatchJsonReader.cs

```diff
@@ -0,0 +1,80 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataBatchJsonReader.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Core

```

Comment:
Classes specific to JSON (light) format should be in Microsoft.OData.Core.JsonLight namespace